amd_openvx_extensions/amd_rpp/source/ExposurebatchPD.cpp

```diff
@@ -132,22 +131,22 @@ static vx_status VX_CALLBACK processExposurebatchPD(vx_node node, const vx_refer
         refreshExposurebatchPD(node, parameters, num, data);
         if (df_image == VX_DF_IMAGE_U8)
         {
-            rpp_status = rppi_exposure_u8_pln1_batchPD_gpu((void *)data->cl_pSrc, data->srcDimensions, data->maxSrcDimensions, (void *)data->cl_pDst, data->exposureValue, data->nbatchSize, data->rppHandle);
+            rpp_status = rppi_exposure_u8_pln1_batchPD_gpu((void *)data->cl_pSrc, data->srcDimensions, data->maxSrcDimensions, (void *)data->cl_pDst, data->exposureValue, data->nbatchSize, data->handle->rppHandle);
```

Comment:
I think it is complaining because data->handle->rppHandle is ambiguous.  (data->handle)->rppHandle and proper typecasting is needed